src/App.tsx

```diff
@@ -35,6 +36,7 @@ const App: React.FC = () => {
               <Route path="/signup" exact component={Signup} />
               <Route path="/home" exact component={Home} />
               <Route path="/settings" exact component={Settings} />
+              <Route path="/dummy" exact component={DummyLeaderboard} />
```

Comment:
Rather than adding this `dummy` leaderboard and having to change things later, I think it makes more sense to initialize the proper container now. You can make this one the `VolunteerLeaderboard` as the `TeamLeaderboard` will come later in Milestone 2.